tensorflow/core/kernels/mkl_requantization_range_per_channel_op.cc

```diff
@@ -73,6 +74,26 @@ class MklRequantizationRangePerChannelOp : public OpKernel {
 
     // Find the ranges of each channel in parallel.
     float out_min_max = std::numeric_limits<float>::min();
+#ifdef ENABLE_MKLDNN_THREADPOOL
+    // TODO: Add eigen parallel_for
+    for(size_t i = 0; i < depth; ++i) {
+      Eigen::Tensor<qint32, 0, Eigen::RowMajor> min =
+          transposed_input.chip<0>(i).minimum();
+      Eigen::Tensor<qint32, 0, Eigen::RowMajor> max =
+          transposed_input.chip<0>(i).maximum();
+      const int32_t min_per_channel = min();
+      const int32_t max_per_channel = max();
+      const int32_t abs_max =
+          std::max(std::abs(min_per_channel), std::abs(max_per_channel));
+      float scale =
+          std::max(std::abs(input_min_data[i]), std::abs(input_max_data[i]));
+      ranges[i] =
+          scale * static_cast<float>(abs_max) / static_cast<float>(1L << 31);
+      if (min_per_channel < 0) is_non_negative = false;
+
+      out_min_max = std::max(out_min_max, ranges[i]);
+    }
+#else
 #pragma omp parallel for reduction(max : out_min_max)
```

Comment:
Same here. Can we wrap `#ifndef ENABLE_MKLDNN_THREADPOOL` around this line instead? Please also keep the `TODO`.

Reply:
Done